bundles/org.openhab.binding.renault/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -94,7 +97,13 @@
 	<channel-type id="batteryavailableenergy">
 		<item-type>Number:Energy</item-type>
 		<label>Battery Energy Available</label>
-		<state pattern="%.1f %unit%" readOnly="true"></state>
+		<state pattern="%.1f kWh" readOnly="true"></state>
```

Comment:
kWh is default unit for energy, I believe - is this change needed?

Reply:
Not sure but I got the converted value in J (for Joules) in my site map page so I added the unit and it displayed the values and unity in kWh which is what I want.   Maybe there is a better way to define the units?

Reply:
Do you also get this with final release version 3.4? There was a fix in core merged into one of the last milestones: openhab/openhab-core#3201.

Reply:
I will test this. I did get this issue openhab:3.4.0.M5.